src/endpoints/getCredentialLinks.ts

```diff
@@ -58,7 +57,7 @@ export const getCredentialLinks: PayloadHandler = async (req, res) => {
             headers: { 'Content-Type': 'application/json' },
             body: JSON.stringify({
                 data: [{ vc: builtCredential, retrievalId: id }],
-                tenantName,
+                tenantName: credential.tenant,
```

Comment:
Do we need to have any flexibility to assign a different tenant name other than the hostname for the admin dashboard deployment?

Are there good reasons why a deployment would not want to use names compatible with the approach?
* Would require character escaping or oddities in setting env vars
* Already have existing tenant names set up in the workflow coordinator that would be a pain to change?

Reply:
https://github.com/skybridgeskills/dcc-workflow-coordinator/pull/1/files